var/spack/repos/builtin/packages/otf2/package.py

```diff
@@ -26,30 +26,21 @@
 from spack import *
 
 
-class Otf2(Package):
+class Otf2(AutotoolsPackage):
     """The Open Trace Format 2 is a highly scalable, memory efficient event
        trace data format plus support library.
     """
 
-    homepage = "http://www.vi-hps.org/score-p"
+    homepage = "http://www.vi-hps.org/projects/score-p"
     url      = "http://www.vi-hps.org/upload/packages/otf2/otf2-1.4.tar.gz"
 
-    version('2.0', '5b546188b25bc1c4e285e06dddf75dfc',
-            url="http://www.vi-hps.org/upload/packages/otf2/otf2-2.0.tar.gz")
-    version('1.5.1', '16a9df46e0da78e374f5d12c8cdc1109',
-            url='http://www.vi-hps.org/upload/packages/otf2/otf2-1.5.1.tar.gz')
-    version('1.4',   'a23c42e936eb9209c4e08b61c3cf5092',
-            url="http://www.vi-hps.org/upload/packages/otf2/otf2-1.4.tar.gz")
-    version('1.3.1', 'd0ffc4e858455ace4f596f910e68c9f2',
-            url="http://www.vi-hps.org/upload/packages/otf2/otf2-1.3.1.tar.gz")
-    version('1.2.1', '8fb3e11fb7489896596ae2c7c83d7fc8',
-            url="http://www.vi-hps.org/upload/packages/otf2/otf2-1.2.1.tar.gz")
+    version('2.0',   '5b546188b25bc1c4e285e06dddf75dfc')
+    version('1.5.1', '16a9df46e0da78e374f5d12c8cdc1109')
+    version('1.4',   'a23c42e936eb9209c4e08b61c3cf5092')
+    version('1.3.1', 'd0ffc4e858455ace4f596f910e68c9f2')
+    version('1.2.1', '8fb3e11fb7489896596ae2c7c83d7fc8')
 
-    def install(self, spec, prefix):
-        configure_args = ["--prefix=%s" % prefix,
-                          "--enable-shared",
-                          "CFLAGS=-fPIC",
-                          "CXXFLAGS=-fPIC"]
```

Comment:
It could be. Could also be old rubbish (commit a411c497abe3dbc0e98f4278a2d5908058802e38).
Hard to tell without more details. I'll put it back in, but it certainly looks strange there (I wish there was a comment in the code).